### PR TITLE
cleaning up a couple things

### PR DESCRIPTION
**Pull request by zfrenchee**

### CODE REVIEW

**Comment by zfrenchee** [comment]
@iamjli I'd like to verify that all of this works for me too with the notebook you provided. Would you point me to the necessary files? I'm mostly interested in map_peaks and TF_regression -- I trust construct_garnet_file

**Comment by iamjli** [comment]
@zfrenchee all the files are specified in the notebook. The relative paths should work as well.

File: .gitignore [hunk 1]
```diff
@@ -7,6 +7,9 @@ garnet_data/reference/ucsc_reference.hg19.tss.bed
 garnet_data/cisBP_hg19_SLIM/garnetDB_cisBP.hg19.LOD.10kb.chr1_SLIM.tsv
 example/regression_plots
 
+example_data/*
```
Comment (iamjli) [inline .gitignore]: Do we need these?

Comment (zfrenchee) [reply .gitignore]: I do =)

**Comment by zfrenchee** [comment]
Thanks!

**Pull request merged**